### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by msasikanth**

### CODE REVIEW

**Comment by msasikanth** [comment]
Current dependencies on/for this PR:
* master
  * **PR #4354** <a href="https://app.graphite.dev/github/pr/simpledotorg/simple-android/4354" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/simpledotorg/simple-android/4354?utm_source=stack-comment).

**Comment by noyize** [comment]
The `sqlite-android version` given is `3.39.` instead of `3.39.2`

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,12 @@
 
 ## Next Release
 
+## 2022-11-28-8523
+
 ### Internal
 
 - Bump Faker to v1.2.9
-- Bump sqlite-android version to 3.39.2
+- Bump sqlite-android version to 3.39.
```
Comment (noyize) [inline CHANGELOG.md:10]: The `sqlite-android version` given is `3.39.` instead of `3.39.2`

**Review by noyize — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (msasikanth) [reply CHANGELOG.md:10]: Good catch

**Review by noyize — APPROVED** [review]

**Pull request merged**